dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -58,23 +64,12 @@ private void testIterable(int expectCount, @Nonnegative ZonedIntervalIterable it
     int actualCount = 0;
     for (ZonedInterval interval : iterable) {
       LOG.debug("Interval is {}", interval);
-      assertOneHourExclusive(interval, iterable.getUnit());
+      Assert.assertEquals(iterable.getDuration(), interval.getDuration());
```

Comment:
Thank you for simplifying this assertion. I would suggest to use the static import for `assertEquals`.